packages/core/src/api/serialization/__snapshots__/serialization.test.ts.snap

```diff
@@ -0,0 +1,73 @@
+
+exports[`Convert simple images to structured HTML > Convert nested image to structured HTML 2`] = `"<div class=\\"_blockGroup_1c48ad\\" data-node-type=\\"blockGroup\\"><div class=\\"_blockOuter_1c48ad\\" data-node-type=\\"block-outer\\"><div class=\\"_block_1c48ad\\" data-node-type=\\"blockContainer\\"><div class=\\"_blockContent_1c48ad\\" data-content-type=\\"simpleImage\\" data-url=\\"exampleURL\\" data-caption=\\"Caption\\" data-width=\\"256\\"><div class=\\"_wrapper_1c48ad\\" style=\\"align-items: flex-start;\\"><div class=\\"_addImageButton_1c48ad\\" style=\\"display: none;\\"><div class=\\"_addImageButtonIcon_1c48ad\\"></div><p class=\\"_addImageButtonText_1c48ad\\"></p></div><div class=\\"_imageAndCaptionWrapper_1c48ad\\"><div class=\\"_imageWrapper_1c48ad\\"><img class=\\"_image_1c48ad\\" src=\\"exampleURL\\" alt=\\"placeholder\\" draggable=\\"false\\" style=\\"width: 0px;\\"><div class=\\"_resizeHandle_1c48ad\\" style=\\"left: 4px;\\"></div><div class=\\"_resizeHandle_1c48ad\\" style=\\"right: 4px;\\"></div></div><p class=\\"_caption_1c48ad\\" style=\\"padding: 4px;\\"></p></div></div></div><div class=\\"_blockGroup_1c48ad\\" data-node-type=\\"blockGroup\\"><div class=\\"_blockOuter_1c48ad\\" data-node-type=\\"block-outer\\"><div class=\\"_block_1c48ad\\" data-node-type=\\"blockContainer\\"><div class=\\"_blockContent_1c48ad\\" data-content-type=\\"simpleImage\\" data-url=\\"exampleURL\\" data-caption=\\"Caption\\" data-width=\\"256\\"><div class=\\"_wrapper_1c48ad\\" style=\\"align-items: flex-start;\\"><div class=\\"_addImageButton_1c48ad\\" style=\\"display: none;\\"><div class=\\"_addImageButtonIcon_1c48ad\\"></div><p class=\\"_addImageButtonText_1c48ad\\"></p></div><div class=\\"_imageAndCaptionWrapper_1c48ad\\"><div class=\\"_imageWrapper_1c48ad\\"><img class=\\"_image_1c48ad\\" src=\\"exampleURL\\" alt=\\"placeholder\\" draggable=\\"false\\" style=\\"width: 0px;\\"><div class=\\"_resizeHandle_1c48ad\\" style=\\"left: 4px;\\"></div><div class=\\"_resizeHandle_1c48ad\\" style=\\"right: 4px;\\"></div></div><p class=\\"_caption_1c48ad\\" style=\\"padding: 4px;\\"></p></div></div></div></div></div></div></div></div></div>"`;
```

Comment:
can we prevent things like the resizehandle from being exported? How would we go about this?

Reply:
Right now we use `display: none` to hide the resize handles, but we could instead unmount them from the DOM. Then they also wouldn't show up in the tests. I think I did it this way originally bc it was simpler and more performant, but adds more clutter to the DOM.

Reply:
Ah also keep in mind that the default image block implements `serialize` and outputs a `figure`. This one is modified to not implement `serialize`, therefore uses the same HTML for the clipboard as for rendering.